kasa/discover.py

```diff
@@ -108,7 +109,12 @@
 # When support for cpython older than 3.11 is dropped
 # async_timeout can be replaced with asyncio.timeout
 from async_timeout import timeout as asyncio_timeout
-from pydantic.v1 import BaseModel, ValidationError
+from mashumaro.config import BaseConfig
+
+try:
+    from mashumaro.mixins.orjson import DataClassORJSONMixin as DataClassJSONMixin
+except ImportError:
+    from mashumaro.mixins.json import DataClassJSONMixin  # type: ignore[assignment]
 
```

Comment:
This feels wrong, is there a nice way around it?

Reply:
This is pretty much the same as what we're doing in `json.py` depending on whether we have orjson installed or not.

Reply:
Yeah.. Hmm, but maybe move it inside `json.py` already, so that we can import `DataClassJSONMixin` in other parts when we start converting other models to use mashumaro?

Reply:
Good point, that's done